assets/src/edit-story/types.js

```diff
@@ -19,6 +19,33 @@
  */
 import PropTypes from 'prop-types';
 
+export const HexPropType = PropTypes.shape({
+  r: PropTypes.number.isRequired,
+  g: PropTypes.number.isRequired,
+  b: PropTypes.number.isRequired,
+  a: PropTypes.number,
+});
+
+export const ColorStopPropType = PropTypes.shape({
+  stop: HexPropType.isRequired,
+  position: PropTypes.number.isRequired,
+});
+
+export const PatternPropType = PropTypes.shape({
```

Comment:
Per note above, I believe, we will also need `a` for the pattern, as the color multiplier.